### ISSUE TITLE
sketchPad.getX() and getY() incorrect if in a panel

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roloww/gui4processing#10]
```
What steps will reproduce the problem?
1. place sketchPad in a panel
2. print out sketchPad.getX() sketchPad.getY()
3. move the panel around

What is the expected output?

Either:
Give position relative to panel (will not change unless sketchpad.moveTo() is 
called but it would still be useful)
Or:
Give current position relative to main window.

Either would be acceptable, the alternative can be calculated.

What do you see instead? unknown negative values

What version of the product are you using? On what operating system?
g4p 3.1.1
processing 2.0b7


Please provide any additional information below.

panel1 = new GPanel(this, 0, 0, 344, 415, "Tab bar text")
sketchPad1 = new GSketchPad(this, 2, 21, 280, 280);

getX() == -170.0
getY() == -186.5

other test sketches get other values, I haven't found what they may relate to.
These do not update when the panel is moved.

This is annoying as the initial position is set relative to the panel, so you 
would expect getX(), getY() to return that even if the value is not updated 
when the parent is moved.

If you put the sketch in the main window then getX() and getY() give correct 
coordinates relative to the window.
getX() and getY() also update correctly if you move the sketch around with 
moveTo()
```

Original issue reported on code.google.com by `yakumo....@gmail.com` on 11 Jan 2013 at 2:18

**Comment by GoogleCodeExporter** [comment]
```
Thhis will be fixed in the next release. 

As a work arround using the current release you can calculate the 
panel-position of the top-left corner of a component on a panel (using your 
example) with

panel1 = new GPanel(this, 0, 0, 344, 415, "Tab bar text")
sketchPad1 = new GSketchPad(this, 2, 21, 280, 280);
panel1.addControl(sketchPad1);
float spX = sketchPad1.getX() + panel1.width/2;
float spY = sketchPad1.getY() + panel1.height/2;

Thanks for reporting it.

```

Original comment by `p.k.la...@gmail.com` on 11 Jan 2013 at 7:35
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
The problem has been fixed and will appear in V3.2.0
```

Original comment by `p.k.la...@gmail.com` on 15 Feb 2013 at 8:07
- Changed state: **Fixed**
